index.html

```diff
@@ -69,30 +96,30 @@ <h3 class="motto" >
                 <ul>
                     <li><a ><img class="twitter icons"src="images\Vector.svg" alt="twitter"></a></li>
                     <li><a><img class="linkedin icons" src="images\Linkedin icon@2x (1).png" alt="linkedin"></a></li>
-                    <li><a><img class="vector-1 icons" src="images\Vector (1).svg" alt=""></a></li>
+                    <li><a><img class="vector-1 icons" src="images\Vector (1).svg" alt="Medium"></a></li>
                     <li><a><img class="vector-2 icons" src="images\Vector (2).svg" alt="github"><a></a></li>
-                    <li><a><img class="vector-3 icons" src="images\Vector (3).svg" alt=""><a></li>
+                    <li><a><img class="vector-3 icons" src="images\Vector (3).svg" alt="AngelList"><a></li>
                 </ul>              
                
               </div>
             </div>
             
     </section>
     </section>
-    
+  </section>
   <!-- finish-mobile-version project starts here-->
-  <section class="works">
+  <!-- <section class="works">
 <div class="card-mobile">
-      <a href=""><img class="img-1" src="images\Snapshoot Portfolio.png" alt=""></a>
+      <a href=""><img class="img-1" src="images\Snapshoot Portfolio.png" alt="Snapshoot-Portfolio"></a>
   <h2>
     Tonic
   </h2>
   <div class="canopy-backend">
   <ul>
   <li class="canopy">CANOPY</li>
-  <li class="counter"><img src="images\Counter.png" alt=""></li>
+  <li class="counter"><img src="images\Counter.png" alt="Counter"></li>
   <li class="back-end">Back End Dev</li>
-  <li class="counter"><img src="images\Counter.png" alt=""></li>
+  <li class="counter"><img src="images\Counter.png" alt="Counter"></li>
   <li class="date">2015</li>
   </ul>
 </div>
```

Comment:
Great job team🤝

- [ ] [Optional] Kindly remove all commented code from your project as this is regarded as a bad practice.